core/src/mindustry/world/meta/values/AmmoListValue.java

```diff
@@ -50,6 +50,10 @@ public void display(Table table){
                     sep(bt, Core.bundle.format("bullet.knockback", Strings.fixed(type.knockback, 1)));
                 }
 
+                if(type.pierce){
+                    sep(bt, Core.bundle.format("bullet.pierce", type.pierceCap == -1 ? "infinite" : type.pierceCap));
```

Comment:
Either "infinite" should be localized as well, or it should just omit the qualifier and say "pierce".

Reply:
Added "infinitepierce" because omitting the qualifier would still leave whitespace there I think?